README.md

```diff
@@ -1,6 +1,8 @@
 # IFTTT-CM
 IFTTT integration with CM.com's apps
 
+[![Build Status](https://travis-ci.com/lVlrChris/CM-IFTTT.svg?branch=master)](https://travis-ci.com/lVlrChris/CM-IFTTT)
```

Comment:
We need to change this to CMTelecom/CM-IFTT, but that is something we can do after this MR.